### ISSUE TITLE
Do you know why the accuracy is lower than the original project

### ISSUE THREAD

**Issue by xmuszq** [auroua/InsightFace_TF#28]
@auroua    E.g., the ACC for LFW in original project could go up to 99.8%. Do you know what's missing here?

**Comment by auroua** [comment]
The author use 100 layer network. Please compare with the `LResNet50E-IR`.
Another reason is the author using a large `batchsize` which I can't use because of the memory limit.

**Comment by siahewei** [comment]
very strange problem? i cannot train well using this project, my best accuracy  96% on lfw, so i changed my training method which based on amloss ,as follows: 1. use the model which had trained well by others. 2. keep inference logit as set trainable=false 2. only train param 'w' in arcface loss part.  but get bad result, the inference loss is alway about 25.   And i have trained "Additive-Margin-Softmax", i get accuracy 99.3% very easy. so  i doubt the  'arc face loss'  really wok?

**Comment by ruobop** [comment]
@siahewei 
I can achieve 99.6% on LFW at 400K iters by simply using this project's original code. I am still waiting for better result with more iters. You may double check your train data and consider increasing batch size a little bit.

**Comment by xmuszq** [comment]
@ruobop  let us know when you get better results. Thanks.

**Comment by HsuTzuJen** [comment]
@auroua In the TF document, It says that using NCHW is better than using NHWC in training mode with CUDA, and NHWC in inference mode.
Is there any way to achieve this?

**Comment by darchonyzx** [comment]
I come into the same problem with @siahewei . I use 96x96 images to train a model, but it seems to up to the bottleneck 0.96 in lfw test.
